server.go

```diff
@@ -386,6 +386,46 @@ func (server *Server) createTLSConfig(entryPointName string, tlsOption *TLS, rou
 	// BuildNameToCertificate parses the CommonName and SubjectAlternateName fields
 	// in each certificate and populates the config.NameToCertificate map.
 	config.BuildNameToCertificate()
+	//Set the minimum TLS version if set in the config TOML
+	if server.globalConfiguration.EntryPoints[entryPointName].TLS.MinVersion != "" {
+		config.PreferServerCipherSuites = true
+		switch server.globalConfiguration.EntryPoints[entryPointName].TLS.MinVersion {
+		case "VersionTLS10":
+			config.MinVersion = tls.VersionTLS10
+		case "VersionTLS11":
+			config.MinVersion = tls.VersionTLS11
+		case "VersionTLS12":
+			config.MinVersion = tls.VersionTLS12
+		}
+	}
+	//Set the list of CipherSuites if set in the config TOML
+	if server.globalConfiguration.EntryPoints[entryPointName].TLS.CipherSuites != nil {
+		config.CipherSuites = make([]uint16, 0)
+		for _, cipher := range server.globalConfiguration.EntryPoints[entryPointName].TLS.CipherSuites {
+			switch cipher {
```

Comment:
@dtomcej as I said on Slack yesterday:
>Let's put this in configuration.go :)

Of course, I would have preferred not redefining these const in traefik, but sadly, this is not possible, even using golang reflect or stringer. Let's keep it simple, and put this in a map in `configuration.go` ;):
```
var cipherSuites = map[string]uint16{
    `TLS_RSA_WITH_RC4_128_SHA`:     tls.TLS_RSA_WITH_RC4_128_SHA,
    `TLS_RSA_WITH_3DES_EDE_CBC_SHA`: tls.TLS_RSA_WITH_3DES_EDE_CBC_SHA,
    ...
}
```